CONTRIBUTING.rst

```diff
@@ -10,9 +10,8 @@
   ``./venv/bin/pip install ../../python-docs-theme/dist/python-docs-theme-2021.8.tar.gz``
   then build the doc using ``make html``).
 - upload it: ``twine upload dist/*``.
-- bump version (YYYY.MM.dev) in setup.py and python_docs_theme/theme.conf
```

Comment:
Do these two files still need to be updated, possibly together with `pyproject.toml` (mentioned at the beginning of the list)?

Reply:
There is no setup.py file and theme.conf doesn't have a version number in it.